### DISCUSSION TITLE
蓝光演唱会 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#100] [模板讨论]
蓝光演唱会 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by xuange-dada** [comment]
Failed at 4/7 request,
Fail assert: {"re": "错误", "from": "content"} from failed_asserts,
Request URL: https://www.lgych.com/wp-content/themes/modown/action/login.php
